src/tst/include/tst/IntegratedFixture.h

```diff
@@ -34,6 +34,18 @@ class IntegratedFixture : public tst::Fixture
   void dumpVerilogAndOdb(const std::string& name) const;
   void removeFile(const std::string& path);
 
+  // Compare with verilog string input
+  // 'remove_file=false' will keep the output verilog for debug
+  void writeAndCompareVerilogOutputString(
```

Comment:
This is fine but eventually I would like VerilogWriter to take an ostream so we can pass an ostringstream and do this in memory.